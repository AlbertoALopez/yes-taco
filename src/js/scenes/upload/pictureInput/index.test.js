import React from 'react';
import PictureInput from './index.js';

test('Dialog renders correctly', () => {
  const wrapper = shallow(<PictureInput />);

  expect(wrapper).toMatchSnapshot();
});
