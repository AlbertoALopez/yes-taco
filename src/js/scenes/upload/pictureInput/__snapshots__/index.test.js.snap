// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component test Should render correctly 1`] = `
<form
  className="picture-form"
  onSubmit={[Function]}
>
  <Row>
    <input
      className="form-input"
      onChange={[Function]}
      type="file"
    />
  </Row>
</form>
`;
